### PR TITLE
Phase2-hgx158 Correct density of PCB for v9 geometry

### PR DESCRIPTION
**Pull request by bsunanda**
A new material with density 1.85 g/cc and composition same a M_NEMA FR4 plate is defined and used for v9 HGCal geometry

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Review by ianna — CHANGES_REQUESTED** [review]

File: Geometry/HGCalCommonData/data/hgcal/v9/hgcal.xml [hunk 1]
```diff
@@ -87,6 +87,23 @@
       <rMaterial name="materials:Copper"/>
     </MaterialFraction>
   </CompositeMaterial>
+  <CompositeMaterial name="HGC_G10-FR4" density="1.85*g/cm3" symbol=" " method="mixture by weight">
```
[the remark this answers is not in the document]
Comment (bsunanda) [reply Geometry/HGCalCommonData/data/hgcal/v9/hgcal.xml]: @ianna I thought about that. Should I then change all scenario's with HGCal?

[the remark this answers is not in the document]
Comment (ianna) [reply Geometry/HGCalCommonData/data/hgcal/v9/hgcal.xml]: yes, please.

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by kpedro88** [comment]
+upgrade

**Comment by ianna** [comment]
+1

**Comment by fabiocos** [comment]
+1

**Pull request merged**